### DISCUSSION TITLE
[High criticity] No distinction made between disbursement and payment

### DISCUSSION THREAD

**Discussion by AlexD-Bearing** [openkfw/TruBudget#871] [General]
No distinction is made in truBudget between "disbursement" and  "payment": 

- "Disbursed amount" refers to the funds transfered from the donor to the budget of partner country. 

- "Payment" refers to the funds which are directly sent to the suppliers or consultants without passing through the budget of the partner country. 

Those two different modes of transfert of fund have real impacts for the the finance ministry and both need to be visible in TruBudget.

**Comment by Stezido** [comment]
The `disburse`-phrase is translated to payment in #571 
Is this change sufficent for the needs of the finance ministry?
